resources/ftbquests/lang/en_us.lang

```diff
@@ -1135,6 +1133,48 @@ q.botania.gourmaryllis.name=Gourmaryllis
+q.botania.campanimia.name=Campanimia
+q.botania.campanimia.desc=This flower efficiently utilizes leftover §dessentia§r from a Thaumcraft cauldron. Each second, it §dsiphons§r essentia from directly attached crucibles, provided there’s space in its §3mana§r buffer.\n\nThe more §dessentia§r it consumes, the greater the §3mana§r output, but it only counts unique aspects. If a single aspect is present in multiple cauldrons, it is still considered one, and will be consumed from all cauldrons. \n\nRemember, the cauldron must be heated to function.
+q.botania.spectrolus.name=Spectrolus
+q.botania.spectrolus.desc=Spectrolus draws §1mana§r by devouring nearby wool, though only wool of a specific color will suffice.\n\nUse the §awand of the forest§r to reveal which color it requires.
+q.botania.nuclianthus.name=Nuclianthus
+q.botania.nuclianthus.desc=A simple botanical reactor that consumes nuclear §4fuel§r compatible with nuclear reactors, generating §3mana§r in return. It also produces §4fuel waste§r as a byproduct. \n\n§4WARNING: If the flower's buffer remains full for too long, it will explode!§r
+q.botania.shulk_me_not.name=Shule Me Not
+q.botania.shulk_me_not.desc=§dShulk Me Not§r is a powerful mana-generating flower. When a shulker strikes another mob near it, both mobs are killed, producing a large amount of mana.
+q.botania.dandelifeon.name=Dandelifeon
+q.botania.dandelifeon.desc=It produces Mana by simulating Conway's Game of Life in a 25×25 area around it. \n\nConsult the Lexica Botania for additional information.
+q.botania.marimorphosis.name=Marimorphosis
+q.botania.marimorphosis.desc=A functional flower that transforms nearby stone into §7Metamorphic Stone§r. The type of §7Metamorphic Stone§r generated varies by biome, producing more biome-specific variants.
+q.botania.amuileria_kaerunea.name=Amuileria Kaerunea
+q.botania.amuileria_kaerunea.desc=This flower activates only during thunder weather. When it detects items nearby that can be charged, it will summon a lightning strike to energize them.
+q.botania.jikanacea.name=Jikanacea
+q.botania.jikanacea.desc=This flower, at the cost of a significant amount of §3mana§r, extends the effect of §1Time in a Bottle§r. The greater the effect of §1TIAB§r, the more mana it consumes. Typically, it can support up to one fully upgraded §1TIAB§r.
+q.botania.rokku_eryngium.name=Rokku Eryngium
+q.botania.rokku_eryngium.desc=This flower is the botanical equivalent of the Astral Sorcery grinding stone. It will absorb nearby astral crystals to cut them down (at the cost of size). The flower don't like giving crystal backs so it needs to be harvested to get cystal back.  \n\nRemember: if the size of the stone is too small, it can still break.
+q.botania.antirrhift.name=Antirrhift
+q.botania.antirrhift.desc=This flower accelerates the process of mana transportation through spreaders, but at the cost of a hefty §420% mana tax§r.
```

Comment:
Replace `%%` instead of `%` everywhere to prevent "Format Error" message.
![image](https://github.com/user-attachments/assets/9a695aaf-5ece-4bcb-9ecd-48e6b5d8b202)